### PR TITLE
Add working flatpak integrations

### PR DESCRIPTION
**Pull request by LyzardKing**
The flatpak now works with some latex editors.

### CODE REVIEW

File: en/installation.md [hunk 1]
```diff
@@ -69,8 +69,10 @@ Depending on your use case and needed integrations it is advisable to choose the
 | Libreoffice \(system\) | ❌ | ❌ | ✅ | ✅ |
 | Libreoffice \(snap\) | ❌ | ❌ | ❌ | ❌ |
 | Libreoffice \(flatpak\) | ❌ | ❌ | ❌ | ❌ |
-| LaTex editor \(system\) | ❌ | ❌ | ✅ | ✅ |
-| LaTeX editor \(snap\) | ❌ | ❌ | ✅ | ✅ |
+| TexShow | ❌ | ✅ | ✅ | ✅ |
+| TexMaker | ❌ | ✅ | ✅ | ✅ |
+| LyX | ❌ | ✅ | ✅ | ✅ |
+| VIm/EMacs | ❌ | ❌ | ✅ | ✅ |
```
Comment (Siedlerchr) [inline en/installation.md:75]:
```suggestion
| Vim/Emacs | ❌ | ❌ | ✅ | ✅ |
```
I guess this is the correct casing?

**Comment by LyzardKing** [comment]
And the casing in the commit message is wrong.. I'm still getting used to the keyboard of a new laptop... the shift key is a bit sticky

**Comment by koppor** [comment]
It is OK for me to remove LaTeX-Editor, because it was not updated since more than 10 years now: http://www.latexeditor.org/news.html

**Comment by LyzardKing** [comment]
@koppor In any case it should not be shown in the linux builds, since it's a windows only editor..

**Pull request merged**